### PR TITLE
Sign Appx files

### PR DESCRIPTION
**Pull request by chcosta**
Fixes https://github.com/dotnet/core-eng/issues/1094

This change enables signing the Appx file.

Validated with a "private official build", that Appx file is being successfully signed with these changes.

@zamont @joshfree @markwilkie

### CODE REVIEW

File: signing/sign.proj [hunk 1]
```diff
@@ -75,6 +75,18 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="SignAppx" DependsOnTargets="GetSignAppxFiles">
```
Comment (zamont) [inline signing/sign.proj]: Out of curiosity, how is this SignAppx target being reached?

File: buildpipeline/Core-Setup-Windows-BT.json [hunk 2]
```diff
@@ -249,6 +249,33 @@
...
+      },
+      "inputs": {
+        "solution": "$(PB_SourcesDirectory)\\signing\\sign.proj",
+        "msbuildLocationMethod": "version",
+        "msbuildVersion": "latest",
+        "msbuildArchitecture": "x64",
+        "msbuildLocation": "",
+        "platform": "$(PB_TargetArchitecture)",
+        "configuration": "$(BuildConfiguration)",
+        "msbuildArguments": "/t:SignAppx $(PB_CommonMSBuildArgs) /v:detailed",
```
Comment (chcosta) [inline buildpipeline/Core-Setup-Windows-BT.json]: This line calls the "SignAppx" target during the official build

**Comment by zamont** [comment]
cc: @nattress

**Comment by joshfree** [comment]
LGTM

**Pull request merged**